packages/block-editor/src/components/inserter/index.js

```diff
@@ -60,6 +60,7 @@ const defaultRenderToggle = ( {
 
 	return (
 		<Wrapper
+			__next40pxDefaultSize={ toggleProps.as ? undefined : true }
```

Comment:
Consumers who are rendering a custom toggle button with the `as` prop will need to take care of this themselves.

Reply:
Same question as https://github.com/WordPress/gutenberg/pull/68157#discussion_r1893661558 re. checking if `toggleProps.as === `Button` ?